common/src/main/java/com/wynntils/core/notifications/MessageContainer.java

```diff
@@ -9,17 +9,24 @@
 
 public class MessageContainer {
     protected TextRenderTask message;
+    private int messageCount;
 
     public MessageContainer(String message) {
-        this.message = new TextRenderTask(message, TextRenderSetting.DEFAULT);
+        this(new TextRenderTask(message, TextRenderSetting.DEFAULT));
     }
 
     public MessageContainer(TextRenderTask message) {
         this.message = message;
+        this.messageCount = 1;
+    }
+
+    public String getOriginalMessage() {
+        return message.getText();
     }
 
     public TextRenderTask getRenderTask() {
-        return message;
+        String messageMultiplier = " §7[x" + this.messageCount + "]";
```

Comment:
You just need to add a 
```
if (messageCount == 1) return message;
```
first, and then it will 1) be correct and 2) be more efficient. ;-)

Reply:
Fixed!